doctrine/mapping_model_classes.rst

```diff
@@ -125,15 +122,22 @@ Annotations, XML, Yaml, PHP and StaticPHP. The arguments are:
     ``DoctrineOrmMappingsPass`` and adapted to use the ``DefaultFileLocator``
     instead of the ``SymfonyFileLocator``::
 
+        use Doctrine\Common\Persistence\Mapping\Driver\DefaultFileLocator;
+        use Doctrine\ORM\Mapping\Driver\XmlDriver;
+        use AppBundle\Model;
+
+        // ...
         private function buildMappingCompilerPass()
         {
-            $arguments = array(array(realpath(__DIR__ . '/Resources/config/doctrine-base')), '.orm.xml');
-            $locator = new Definition('Doctrine\Common\Persistence\Mapping\Driver\DefaultFileLocator', $arguments);
-            $driver = new Definition('Doctrine\ORM\Mapping\Driver\XmlDriver', array($locator));
+            $locator = new Definition(DefaultFileLocator::class, array(
+                array(realpath(__DIR__ . '/Resources/config/doctrine-base')),
+                '.orm.xml'
+            ));
+            $driver = new Definition(XmlDriver::class, array($locator));
 
             return new DoctrineOrmMappingsPass(
                 $driver,
-                array('Full\Namespace'),
+                array(Model::class),
```

Comment:
Yes, decided to use a more real example than `Full\Namespace`. Also, because `::class` always is a full namespace.